packages/api/src/platforms/vtex/index.ts

```diff
@@ -36,6 +36,7 @@ export interface Options {
   channel: string
   locale: string
   hideUnavailableItems: boolean
+  simulationBehavior?: 'default' | 'skip' | 'only1P'
```

Comment:
**Question:** Are we providing any guidance on when to use each option?

Reply:
Not now, we can think about adding something on our doc. But that is something related to Search ([ref](https://developers.vtex.com/docs/api-reference/intelligent-search-api#get-/product_search/-facets-?endpoint=get-/product_search/-facets-)).